x509/build/fmc_alias_cert_tbs.rs

```diff
@@ -0,0 +1,212 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    fmc_alias_cert_tbs.rs
+
+Abstract:
+
+    File contains generated FMC Alias Certificate template
```

Comment:
Make it clear that this is autogenerated by a particular tool, and make it clear to the reader how to regenerate.

Reply:
Done